.github/workflows/elixir-ignored.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     paths-ignore:
       - '.github/workflows/elixir.yml'
+      - '.github/workflows/elixir-ignored.yml'
```

Comment:
If we paths-ignore this file, that'll mean that if we create a PR that changes `only` this file, the PR will block as Elixir workflow is required.

Reply:
Hmm, really good point

Reply:
removed it.